CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 
 ## v5.0.0 (WIP)
 
+- Enables setting specific origins for CORS via the
+  WHARF_HTTP_CORS_ALLOWORIGINS environment variable. This is to make sending
+  `Authorization` headers possible. (#101)
+
```

Comment:
Suggest rephrasing this to begin with "Added", perhaps also copy the format used in some other logs, such as:

> ```md
> - Added configuration for Sqlite file path, environment variable `WHARF_DB_PATH`
>   or the YAML key `db.path`. Defaults to `wharf-api.db`. (#86)
> ```

```suggestion
- Added configuration of specific origins for CORS via the environment variable
  `WHARF_HTTP_CORS_ALLOWORIGINS` or the YAML key `http.cors.allowOrigins`. This
  is to make sending `Authorization` headers possible. (#101)
```